build/sign.proj

```diff
@@ -34,14 +30,7 @@
     <Exec Command="$(PowerShell) -NonInteractive -NoLogo -NoProfile -Command &quot;$(RemoveSignedUnsignedFolders)&quot;" ContinueOnError="WarnAndContinue" IgnoreExitCode="true" />
   </Target>
 
-  <Target Name="BeforeAll">
-    <Message Importance="high" Text="Sign: BeforeAll started" />
-    <PropertyGroup>
-      <DisableSigning>true</DisableSigning>
-    </PropertyGroup>
-  </Target>
-
-  <Target Name="All" DependsOnTargets="BeforeAll;netstandard20">
+  <Target Name="All" DependsOnTargets="netstandard20">
```

Comment:
You can remove the DisableSigning logic. That only existed to do the batch processing. So, this can simply be:
```suggestion
  <Target Name="All" DependsOnTargets="netstandard20" />
```
All the rest of the target stuff can be removed.